feature/page_objects/base_page.py

```diff
@@ -19,4 +22,13 @@ def wait_until_element_is_clickable(self, css_selector):
     WebDriverWait(self.driver, 20).until(expected_conditions.element_to_be_clickable(By.CSS_SELECTOR, css_selector))
 
 def wait_until_text_is_present(self, css_selector, text):
-    WebDriverWait(self.driver, 20).until(expected_conditions.text_to_be_present_in_element((By.CSS_SELECTOR, css_selector), text))
\ No newline at end of file
+    WebDriverWait(self.driver, 20).until(expected_conditions.text_to_be_present_in_element((By.CSS_SELECTOR, css_selector), text))
+
+def wait_until_alert_is_present(self):
+    WebDriverWait(self.driver, 20).until(expected_conditions.alert_is_present())
+    
+def get_alert_text(self):
+    return self.driver.switch_to.alert.text
+
+def get_uuid4():
```

Comment:
This function could be something more representative of what it is doing.
for example `create_unique_user()`.
That way you can remember what is the purpose of the function